# Foreword  {.unnumbered}

This textbook and companion course was conceived in 2020 during the depths of the global coronavirus pandemic. 
James Fellows Yates and I were discussing the need for more training resources in the emerging field of ancient metagenomics, and, together with Pierre Stallforth at the Leibniz HKI, I had just received generous funding from the Werner Siemens Foundation to advance new techniques and methodologies in the emerging research area of microbial paleobiotechnology. 
Trapped at home and interacting with the world primarily through screens, we began to think seriously about how we might harness our new digital connectivity and our new funding to expand computational training opportunities, irrespective of nationality or institution, for a new generation of students eager to pursue a career in the young field we and others were striving to build: ancient metagenomics.  

When I began graduate school in 2004, the idea that one day we would be able to reconstruct the genomes and ecological communities of microbes that lived thousands or even millions of years ago seemed a remote possibility. 
This was still the era of PCR and Sanger sequencing, and each newly reported microbial ancient DNA sequence was met with deep skepticism and vigorously debated. 
By decade’s end, however, emerging high throughput sequencing technologies began to change that outlook, and in 2011 the first complete ancient microbial genome was published. 
The study of ancient microbes started to gain momentum, and after another decade of research starts and stops, false turns, dead ends, and dramatic breakthroughs, the seemingly impossible was starting to happen - we were developing an archaeology of microbes. 
As the field grew, so did its applications–expanding beyond its origins in pathogen genomics to include microbiome studies, fermented food studies, clinical metagenomics, and environmental microbial ecology. 

Around this time, people finally stopped asking why studies of ancient microbes belonged in the fields of archaeology and anthropology and started understanding how fundamental microbes are to the long arc of human health, diet, and culture. 
In 2019, I co-organized a Wenner Gren Symposium on _Cultures of Fermentation_, an interdisciplinary meeting that brought together social anthropologists, biological anthropologists, and archaeologists to examine the role of microbes in human cultures and societies from the Paleolithic to the present, and we won the 2023 American Anthropological Association’s Cross-Fields Prize for our companion article about the symposium in _Current Anthropology_. 
It was a milestone in gaining recognition for the field and the first major anthropological prize awarded for the study of microbes.

Microbial ancient metagenomics was finally coming into its own, and a vibrant new research community was taking shape. 
Much of this community centers around a group affectionately known as SPAAM (https://spaam-community.org), which stands for Standards Precautions and Advances in Ancient Metagenomics. 
It was the first in what is now a long list of cheeky food-themed acronyms in biomolecular archaeology, which includes the tools MALT (MEGAN Alignment Tool) and HOPS (Heuristic Operations for Pathogen Screening), as well as other ancient biomolecules interest groups in general like HAAM (Human Ancient DNA, Ancestry, and Mobility) and PAASTA (Palaeoproteomics And Archaeology, Society for Techniques and Advances). 

Alexander Herbig and Johannes Krause organized the first SPAAM conference in 2016, and its first major output was a 2017 article in the journal Annual Review of Genomics and Human Genetics in which we proposed a research framework for the emerging field of microbial archaeology. 
Recognizing the potential for SPAAM to serve as a hub for fostering education, networking, and collaborative research in ancient metagenomics, James Fellows Yates built upon this initial effort by further organizing this nascent community of researchers and students around the world, coordinating subsequent SPAAM conferences, and building up resources and workshops to support the far-flung members of this growing field. 
In 2022, SPAAM joined the International Society for Biomolecular Archaeology (ISBA) as an affiliate group. 

Reflecting on our own trajectories of education and training, James and I had been talking for some time about how our field needed a more formalized way of training new researchers in the field and getting students up to speed on the very quickly evolving methods, concepts, and especially bioinformatics strategies that underlie our work. 
The Werner Siemens Foundation grant I had received included funds for education and outreach, and James and I hatched the idea of developing an online ancient metagenomics course to further these goals. 
The Jena School for Microbial Communications (JSMC) at Friedrich Schiller University signed on to accredit the course, as did the Max Planck-Harvard Research Center for the Archaeoscience of the Ancient Mediterranean (MHAAM). 
More than a dozen group leaders, postdocs, and grad students at the MPI-EVA and other institutions volunteered to help develop the course, and in 2022 we launched our first Introduction to Ancient Metagenomics Summer School, a week-long intensive online practical course open to students worldwide for free. 

From the beginning, we had the idea of making all of the teaching material for the course available on a website, and the current textbook grew organically out of our experiences teaching the material to students with diverse educational backgrounds and skill levels. 
The book is written with a target audience in mind of a Master’s student or early stage PhD student who is highly motivated to begin work in the field of ancient metagenomics, but who may be transitioning from a different field, whether that is biology, computer science, or even archaeology. 
We aim to provide both a broad overview and a solid foundation in the most essential concepts needed to begin work in ancient metagenomics, and we encourage our readers to follow up with the citations provided in the textbook to gain deeper understanding of the many topics we cover in an abbreviated form. 

First and foremost, we wanted this textbook to be hands-on and practical, and so we introduce not only theoretical concepts but also provide datasets, pipelines, and code to allow students to go through the full process of analyzing ancient microbial DNA datasets–from fastq files to phylogenies and functional genomics. 

This textbook is a labor of love produced through the efforts of many people who have generously volunteered their time to create a resource for future generations, to help give new students a jumpstart in the field, and to promote a research community that is collaborative, supportive, and committed to advancing knowledge and understanding of the microbial world around us and how it came to be.

_Christina Warinner_
_Harvard University, Cambridge, Massachusetts, USA, May 2024_

## Thank you

We would like to extend a special thank you to the following people who contributed to the development of the course and companion textbook:

- Aida Andrades Valtueña
- Maxime Borry
- Keri Burge
- Remí Densie
- Sebastian Duchene
- Jasmin Frangenberg
- Meriam Guellil
- Alexander Herbig
- Alina Hiß
- Alexander Hübner
- Arthur Kocher
- Denise Kühnert
- Thiseas Lamnidis
- Megan Michel
- Nikolay Oskolkov
- Betsy Nelson
- Gunnar Neumann
- Kevin Nota
- Aleksandra Laura Pach
- Vilma Pérez
- Clemens Schmid
- Irina Velsko
- Robin Warner
- Christina Warinner
- James Fellows Yates
- Guilia Zampirolo
- Tessa Zeibig

## Acknowledgements

We are very grateful to Matthias Meyer and Pierre Stallforth for generously reading our early drafts of sections of our textbook and providing valuable feedback and suggestions.  