---
title: Ancient Metagenomics
---

The techniques in this section of the book can be used in a variety of stages of any ancient metagenomics projects, for screening for pathogens (what species should I target for downstream genomic mapping?), for differential abundance analysis (does the community make of this sample change between different cultural periods?), but also for reference-free assembly of genomes (can I recover the genome architecture of a variety of species in my sample?). It focuses on the concept of 'many samples to many genomes' using high-throughput techniques and algorithms and trying to analyse data at whole 'community' levels.

## [Taxonomic Profiling](taxonomic-profiling.qmd)


<!-- TODO -->

_TBC_


## [_De novo_ Assembly](denovo-assembly.qmd)

_De novo_ assembly of ancient metagenomic samples enables the recovery of the genetic information of organisms without requiring any prior knowledge about their genomes. Therefore, this approach is very well suited to study the biological diversity of species that have not been studied well or are simply not known yet.

In this chapter, we will show you how to prepare your sequencing data and subsequently _de novo_ assemble them. Furthermore, we will then learn how we can actually evaluate what organisms we might have assembled and whether we obtained enough data to reconstruct a whole metagenome-assembled genome. We will particularly focus on the quality assessment of these reconstructed genomes and how we can ensure that we obtained high-quality genomes.

## [Authentication](authentication.qmd)


<!-- TODO -->

_TBC_

## [Contamination](contamination.qmd)


<!-- TODO -->

_TBC_

